edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -2849,7 +2921,17 @@ public void applyDropScores(final Collection<AssignmentGradeRecord> gradeRecords
                     final List<AssignmentGradeRecord> gradesByCategory = new ArrayList<>();
                     final List<AssignmentGradeRecord> gradeRecordsByCatAndStudent = gradeRecordMap.get(studentId + cat.getId());
                     if(gradeRecordsByCatAndStudent != null) {
-                        gradesByCategory.addAll(gradeRecordsByCatAndStudent);
+                        for(AssignmentGradeRecord agr : gradeRecordsByCatAndStudent){
+                            boolean excluded;
```

Comment:
BooleanUtils.toBoolean

Reply:
All reviewed changes addressed in latest commit.